linode/manager
# feat: [M3-6077] - Add Maintenance Mode Environment Variable and Clean Up `<MaintenanceScreen />`

## Description 📝

- Adds `REACT_APP_ENABLE_MAINTENANCE_MODE` to allows us to explicitly enable maintenance mode
- Cleans up  and modernizes `<MaintenanceScreen />`
- Makes the 🔧 a better color so it looks good in dark mode
- Removes the word `scheduled` from the copy because it may not always be true

## Preview 📷

![Screenshot 2023-05-24 at 2 06 37 PM](https://github.com/linode/manager/assets/115251059/5f4aebf0-d510-443c-a046-20079dc3b42a)

## How to test 🧪
- Checkout this PR
- Spin up dev server with `yarn up` or `yarn dev`
- Open Cloud Manager to any page
- Add `REACT_APP_ENABLE_MAINTENANCE_MODE=true` to your `packages/manager/.env`
- Verify your Cloud Manager refreshed and went into maintenance mode
- Comment out `REACT_APP_ENABLE_MAINTENANCE_MODE=true`
- Verify that Cloud Manager is no longer in maintenance mode
